specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2024-09-01-preview/appconfiguration.json

```diff
@@ -0,0 +1,3168 @@
+      "properties": {
+        "value": {
+          "description": "The collection value.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApiKey"
+          }
+        },
+        "nextLink": {
+          "description": "The URI that can be used to request the next set of paged results.",
+          "type": "string"
+        }
+      }
+    },
+    "ApiKey": {
+      "description": "An API key used for authenticating with a configuration store endpoint.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The key ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "A name for the key describing its usage.",
+          "type": "string",
+          "readOnly": true
+        },
+        "value": {
+          "description": "The value of the key that is used for authentication purposes.",
+          "type": "string",
+          "readOnly": true
+        },
+        "connectionString": {
+          "description": "A connection string that can be used by supporting clients for authentication.",
+          "type": "string",
+          "readOnly": true
+        },
+        "lastModified": {
+          "format": "date-time",
+          "description": "The last time any of the key's properties were modified.",
+          "type": "string",
+          "readOnly": true
+        },
+        "readOnly": {
+          "description": "Whether this key can only be used for read operations.",
+          "type": "boolean",
+          "readOnly": true
+        }
+      }
+    },
+    "RegenerateKeyParameters": {
+      "description": "The parameters used to regenerate an API key.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The id of the key to regenerate.",
+          "type": "string"
+        }
+      }
+    },
+    "GenerateSasTokenParameters": {
+      "description": "Parameters used for generating SAS token.",
+      "required": [
+        "resourceType",
+        "exp",
+        "sasKind"
+      ],
+      "type": "object",
+      "properties": {
+        "resourceType": {
+          "description": "The data plane resource type that the SAS token is authorized to access.",
+          "type": "string",
+          "enum": [
+            "Kv",
+            "Snapshot"
+          ],
+          "x-ms-enum": {
+            "name": "resourceType",
+            "modelAsString": true
+          }
+        },
+        "signedParameters": {
+          "description": "The parameters to define the data plane resource scope.",
+          "$ref": "#/definitions/SignedParameters"
+        },
+        "exp":{
+          "description": "The time that the SAS token expires in the Universal ISO 8601 DateTime format. Max allowed expiration is 1 year from the time of token creation.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "cch":{
```

Comment:
What is cch? cache control header? Is there a reason to abbreviate?

@avanigupta

Reply:
Yes, this is a field in the SAS token value. Abbreviated to keep the token as short as possible. Storage uses [rscc ](https://learn.microsoft.com/en-us/rest/api/storageservices/create-user-delegation-sas#specify-query-parameters-to-override-response-headers) for this field.

Reply:
It seems to me that for the RP API this should either be expanded to `cacheControlHeader` or `cacheControlMaxAge`. Out of the two, I think cacheControlMaxAge would be more helpful. And if there's an agreeance on that I would suggest changing the sas token parameter key to `ccma`

Reply:
cacheControlMaxAge and ccma sound good